src/iosrv/init.c

```diff
@@ -627,6 +654,7 @@ server_init(int argc, char *argv[])
 
 	return 0;
 
+	crt_unregister_event_cb(dss_crt_event_cb, NULL);
```

Comment:
Hmm, this should also be done in server_fini()?

Reply:
Oh, good catch. Just realized that this was a dumb mistake; will never be called here.